csfieldguide/static/interactives/scene-editor/js/scene-editor.js

```diff
@@ -124,13 +124,31 @@ $(document).ready(function () {
   $("#apply-transformation").click(applyTransformation);
   $("#delete-button").click(deleteSuspect);
 
-  $('.matrix-row input').on('keyup bind cut copy paste', validateInput);
-  $('.vector-row input').on('keyup bind cut copy paste', validateInput);
+  $('.matrix-row input').on('keyup bind cut copy', function() {
+    validateInput($(this));
+  });
+  $('.vector-row input').on('keyup bind cut copy', function() {
```

Comment:
Consider setting these to on `change` because it's not validating all of the input boxes on a paste event

Alternatively, at the end of your `populateInputsFromPaste` function, add:
```
  // Run the error-check
  $('.matrix-row input').trigger('keyup');
  $('.vector-row input').trigger('keyup');
```

I think i'd do the latter